### ISSUE TITLE
chore: add `Português - pt-BR` language support

### ISSUE THREAD

**Issue by brionmario** [asgardeo/web-ui-sdks#159]
### Current Limitation

Currently, the Asgardeo JavaScript SDK does not have support for **Portuguese (Brazil - pt-BR)**. Users who prefer Brazilian Portuguese will not see UI components translated, limiting accessibility for Portuguese-speaking users.

> [!Important]
> Contributors for this task should be **native Portuguese (Brazilian) speakers or have professional fluency** to ensure accurate, culturally appropriate translations.

### Suggested Improvement

Add **Portuguese (`pt-BR`) language support** to the `@asgardeo/i18n` package.

**Implementation Plan:**

1. Create a new locale file:

    Follow https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md#contributing-to-the-internalization-i18n-package

3. Add documentation for usage:  
   - Update the SDK docs: https://wso2.com/asgardeo/docs/sdks/react/overview/

### Please select the package issue is related to

@asgardeo/react, @asgardeo/nextjs

### Version

current

### Reporter Checklist

- [x] I have searched the existing issues and this is not a duplicate.
- [x] I have provided all the necessary information.
- [x] I have verified the improvement is not available in the latest version of the package.

**Comment by adityagupta0251** [comment]
Hi @brionmario 👋,

I’m interested in contributing to this issue for Hacktoberfest.  
Although my primary languages are Hindi, English, and other Indian regional languages, I am willing to learn and accurately translate the SDK UI components into Brazilian Portuguese (pt-BR) following the contribution guidelines.  

Could you kindly assign this issue to me? 🙏

**Comment by sagara-gunathunga** [comment]
@fosbsb given you are a native Português speaker I'll assign this issue to you. For the visibility please comment here with the link of your PR.  

Also, as mentioned in the [WSO2 Hacktoberfest ](https://wso2.com/hacktoberfest/)web page, only fill the form once your PR is merged so that we can allocate the credits. If you need any assistance please comment here.
